docs/usermanual.md

```diff
@@ -544,3 +545,12 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the IOT Agent (as explained [here](installationguide.md#usage) and then you
+can access to:
+
```

Comment:
I'd suggest to add a sentence about which API is being covered by the swagger documentation. Something like this:

> The swagger documentation provided at /api-docs covers the HTTP southbound interface of the agent. The northbound HTTP interface is not covered.

Reply:
fixed in 2c34c9ab082a7e0b43255d21f690ffed0157703d